ja-jp/Content/index-admin.htm

```diff
@@ -304,7 +304,7 @@
                                             <div class="body-main">
                                                 <!-- Main content body -->
                                                 <div role="main" id="mc-main-content">
-                                                    <h1 style="margin-top:35px;">Okta管理者用ドキュメント</h1>
+                                                    <h1 style="margin-top:35px;">Okta Classic Engine</h1>
```

Comment:
Is `Okta Classic Engine` a product name that should never be translated? If that is the case, we need to let Peng or Yasmin know

Reply:
Yes that's right, product name, Do Not Translate.
I changed that here myself.